packages/metadata-common/src/MetadataEditor.tsx

```diff
@@ -110,6 +110,16 @@ export class MetadataEditor extends ReactWidget {
     this.update();
   }
 
+  private isFieldEmpty(schemaField: any): boolean {
+    return (
+      schemaField === undefined ||
+      schemaField === null ||
+      schemaField === '' ||
+      schemaField === [] ||
+      schemaField === '(No selection)'
+    );
+  }
```

Comment:
The way this is implemented, it's not necessary to be a class method but works.  I figured it would act on `this.metadata` and take the field name as a parameter.  

If we could at least change the parameter name to `schemaValue` then that might eliminate some potential confusion.  But then the question arises - should this be named `IsValueEmpty()`?  :smile:

Reply:
this would be a good function to put in a util, but I didn't think it was worth creating a util.ts just for this function (thus making it a private class method).

As for naming, you are correct, based on my current implementation it should be value not field. I'll push an update promptly